src/main/java/com/cloudant/client/org/lightcouch/CouchDbClient.java

```diff
@@ -89,11 +56,8 @@
  * .replicator()}
  * <p/>
  * <p/>
- * <p>At the end of a client usage; it's useful to call: {@link #shutdown()} to ensure proper
- * release of resources.
  *
  * @author Ahmed Yehia
- * @see CouchDbClientAndroid
  * @since 0.0.2
  */
 public class CouchDbClient extends CouchDbClientBase {
```

Comment:
Yes, that's a good idea.  I wasn't completely clear of the purpose of this class and it seemed to add obscurity.